### PR TITLE
Instructions for gh-pages regeneration

### PR DESCRIPTION
**Pull request by SergeyKanzhelev**
#84

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
@jjjordanmsft is this how you did it last time? I was thinking - maybe it will be easier to use `/docs` folder in master branch instead?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,6 +13,20 @@ Run `python setup.py sdist`
...
 
+## Generate documentation
+
+1. Install [sphinx](http://www.sphinx-doc.org/en/master/usage/installation.html): `brew install sphinx-doc`.
+2. Open docs folder `cd docs`
+3. Generate html documents: `make html`
+4. Update gh-pages branch
+    ``` bash
+    git clone https://github.com/Microsoft/ApplicationInsights-Python.git python-gh-pages
+    cd python-gh-pages/
```
Comment (jjjordanmsft) [inline CONTRIBUTING.md]: Perhaps this is obvious, but you also need to switch to the `gh-pages` branch here.

File: doc/conf.py [hunk 2]
```diff
@@ -52,7 +52,7 @@
 
 # General information about the project.
 project = 'Application Insights SDK for Python'
-copyright = '2014, Microsoft'
+copyright = '2018, Microsoft'
```
Comment (jjjordanmsft) [inline doc/conf.py]: IANAL but shouldn't this be 2014-2018?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -13,6 +13,20 @@ Run `python setup.py sdist`
 
 Use `pip install -e <path>` to install it into test application for the testing purposes.
 
+## Generate documentation
+
+1. Install [sphinx](http://www.sphinx-doc.org/en/master/usage/installation.html): `brew install sphinx-doc`.
```
Comment (jjjordanmsft) [inline CONTRIBUTING.md]: brew is Macintosh specific 👎 .

The cross-platform method is to make a virtualenv and install sphinx into it:
```bash
% python3 -m venv env       # Create 'env' directory
% . env/bin/activate        # Activate virtual environment
% pip install sphinx django # Install sphinx and django (so sphinx can find it)
% python setup.py install   # Install applicationinsights package
% cd doc
% make html
.
.
.
% deactivate   # Exit virtualenv
```

**Comment by jjjordanmsft** [comment]
re: docs vs gh-pages.  Maybe it's better -- I don't have a strong opinion either way.

**Pull request merged**